Intern.Core.Api.Tests.Unit/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
@ADILATIC Remove this file. In the VS add project dialog there is a setting you need to uncheck, this file is automatically created. We do not use Global usings.

Reply:
@ADILATIC All you need to do now is delete it and commit the change on this branch.